terratest/test/k8gb_ingress_mandatory_http_test.go

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
We already have it covered in terratest/test/k8gb_basic_app_test.go

Reply:
@k0da , I see, amending....